pom.xml

```diff
@@ -1230,6 +1232,25 @@ under the License.
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-remote-resources-plugin</artifactId>
+          <executions>
+            <execution>
+              <id>site-download</id>
+              <goals>
+                <goal>process</goal>
+              </goals>
+              <phase>none</phase>
```

Comment:
I would rather bind this to `pre-site`

Reply:
due to inheritance we can put here pre-site